src/components/m-header-navigation/js/header-navigation.js

```diff
@@ -2,6 +2,7 @@ import { add, remove } from '../../../js/class-list';
 import { requestAnimationFrame } from '../../../js/request-animation-frame';
 import forceRepaint from '../../../js/force-repaint';
 import UiEvents from '../../../js/ui-events';
+import isEdge from '../../../js/is-edge';
```

Comment:
please add it in the shame.js

Reply:
hmh, let's rename the file to `is-edge-SHAME.js`?

Reply:
or we could also introduce a folder called `shame/` to group all shame code

Reply:
i would put all the shame stuff in one file, very dirty so that it looks bad

Reply:
i hope that we remove that code soon

Reply:
thats ok with the folder as well

Reply:
Okidok, I will move it into a separate folder.

Reply:
thanks :)

Reply:
Done :)